src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Similar to the previous comment, replace the hardcoded value for the number of days and the discount with constants. This will improve the clarity of the code and make it easier to understand the significance of these values.